### PR TITLE
WBCAMS-151 occupational interest filter drop down and tooltip

### PR DESCRIPTION
**Pull request by rjkuyvenhoven**
iPhone-13-v16-3

### CODE REVIEW

File: src/web/themes/custom/workbc/js/interactive-map.js
Comment (infojunkie) [inline src/web/themes/custom/workbc/js/interactive-map.js:1]: I don't think this file needs to be committed

Comment (rjkuyvenhoven) [reply src/web/themes/custom/workbc/js/interactive-map.js:1]: It doesn't, the file has been removed from the PR

**Review by infojunkie — APPROVED** [review]
Lgtm, thanks! Great job solving this puzzle.

**Comment by rjkuyvenhoven** [comment]
had some help from you and Jonathan

**Pull request merged**